projects/ion/src/lib/tag/tag.component.html

```diff
@@ -0,0 +1,15 @@
+<div
+  data-testid="ion-tag"
+  [style.color]="!context && validateColor()"
+  [style.background]="!context && validateColor() + '1A'"
+  [class]="setTagType()"
+>
+  <ion-icon
+    data-testid="tag-icon"
+    class="tag-icon"
+    size="15"
```

Comment:
Icon size is 12.

<img width="487" alt="image" src="https://user-images.githubusercontent.com/6165180/184896326-e898a894-6ac0-422b-a3d7-82374ac57384.png">

Reply:
In the implementation leaving the icon at 12px made it to small
12px
![image](https://user-images.githubusercontent.com/23461020/185211318-078e1d33-9128-443b-bd78-d30fdceed85a.png)
15px
![image](https://user-images.githubusercontent.com/23461020/185211435-2c3b0a03-2794-4865-bdf8-ff71fb25f9ca.png)

should i put 12 or 15?